build.xml

```diff
@@ -559,4 +559,4 @@
     </sequential>
   </target>
 </project>
-<!-- :tabWidth=2: -->
\ No newline at end of file
+<!-- :tabWidth=2: -->
```

Comment:
Sorry about this. The change was made using the GitHub editor, and this is just something it did.

If it's a problem, I can undo it...